test/androidTest/java/org/thoughtcrime/securesms/espresso/Helper.java

```diff
@@ -14,4 +14,49 @@ public Helper(ActivityTestRule<ConversationListActivity> activityRule) {
     public ConversationsHelper goConversations() {
         return new ConversationsHelper(new HelperSecret());
     }
+
+    public PreferencesHelper resetNickname() {
+        return new PreferencesHelper(new HelperSecret());
+    }
+
+    public PreferencesHelper setNickname(String message) {
+        return new PreferencesHelper(new HelperSecret());
+    }
+
+    /* ASSERTIONS */
```

Comment:
All of these assertions are already in BaseHelper, I imagine this is from the merge, but please remove em all.